test/dynamicscrm/metadata/main.go

```diff
@@ -5,18 +5,16 @@ import (
 	"fmt"
 	"os"
 	"os/signal"
-	"strings"
 	"syscall"
-	"time"
 
 	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/common/handy"
 	connTest "github.com/amp-labs/connectors/test/dynamicscrm"
 	"github.com/amp-labs/connectors/test/utils"
-	"github.com/amp-labs/connectors/test/utils/mockutils"
 )
 
 var (
-	objectName = "accounts"
+	objectName = "contacts"
```

Comment:
Contacts is problematic it has `subscriptionid` which is not valid for read.

IsValidForRead = False